app/src/main/java/com/duycuong/weather/ui/screen/main/MainViewModel.java

```diff
@@ -58,9 +61,9 @@ public void onStartLoading() {
 
             @Override
             public void onGetSuccess(AddressResponse addressResponse) {
-                String address = addressResponse.getResults().get(0).getAddressComponents()
+                  mAddress = addressResponse.getResults().get(0).getAddressComponents()
```

Comment:
ERROR: 'method def' child have incorrect indentation level 18, expected level should be one of the following: 16, 20.